DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json

```diff
@@ -0,0 +1,95 @@
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "CrowdStrikeFalconEventStream",
+        "baseQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\""
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Hosts with Detections",
+        "query": "CrowdStrikeFalconEventStream \n | where EventType == \"DetectionSummaryEvent\" \n| summarize count() by DstHostName \n | top 10 by count_"
+    }, {
+        "description": "Top 10 Users with Detections",
+        "query": "CrowdStrikeFalconEventStream \n | where EventType == \"DetectionSummaryEvent\" \n| summarize count() by DstUserName \n | top 10 by count_"
+    }],
+    "connectivityCriterias": [{
+        "type": "IsConnectedQuery",
+        "value": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\"\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"
+    }],
+    "dataTypes": [{
+        "name": "CommonSecurityLog (CrowdStrikeFalconEventStream)",
+        "lastDataReceivedQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\"\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+    }],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [{
+            "provider": "Microsoft.OperationalInsights/workspaces",
+            "permissionsDisplayText": "read and write permissions are required.",
+            "providerDisplayName": "Workspace",
+            "scope": "Workspace",
+            "requiredPermissions": {
+                "read": true,
+                "write": true,
+                "delete": true
+            }
+        }, {
+            "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+            "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+            "providerDisplayName": "Keys",
+            "scope": "Workspace",
+            "requiredPermissions": {
+                "action": true
+            }
+        }]
+    },
+    "instructionSteps": [
+        { 
+            "title": "", 
+            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow the steps](https://aka.ms/sentinel-crowdstrikefalcon-parser) to use the Kusto function alias, **CrowdStrikeFalconEventStream**",
```

Comment:
Please use https://aka.ms/sentinel-crowdstrikefalconendpointprotection-parser for parser redirection link.

Reply:
resolved